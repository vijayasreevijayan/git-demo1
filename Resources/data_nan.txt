1,13.4,19.3
4.1,15.55,2
5, 8, 9.12, 5, 6, 8
dfj, d, 9