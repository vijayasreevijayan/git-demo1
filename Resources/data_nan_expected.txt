data_nan.txt
Line 1 average 11.2333 standard deviation 9.34041
Line 2 average 7.21667 standard deviation 7.29286
Line 3 average 6.85333 standard deviation 1.75302
